### PR TITLE
Rebrand SCAN RoR as plain RoR

### PR DESCRIPTION
**Pull request by kairstenfay**
We are now moving the branding for SCAN under the SFS umbrella. Call the
RoR workflow, simply, "return of results" or "SFS RoR", and drop the
SCAN language.

The new view for results is `shipping.return_results_v3`. There are
currently no results outside of the SCAN IRB study to return.

Depends on https://github.com/seattleflu/id3c-customizations/pull/136

### CODE REVIEW

File: bin/return-of-results/generate-results-csv [hunk 1]
```diff
@@ -20,10 +20,10 @@ main() {
         esac
     done
 
-    # Change to the ./bin/scan-return-of-results directory in the backoffice checkout
+    # Change to the ./bin/return-of-results directory in the backoffice checkout
     cd "$(dirname "$0")"
 
-    ./transform <(./export-redcap-scan) <(./export-id3c-scan-return-results)
+    ./transform <(./export-redcap-scan) <(./export-id3c-return-results)
```
Comment (kairstenfay) [inline bin/return-of-results/generate-results-csv:26]: Hmm. Realizing now that we may need multiple `export-redcap` scripts. This section remains a TODO

**Comment by kairstenfay** [comment]
[Update return of results workflow for UW Reopening](https://trello.com/c/ZEib91cE/762-update-return-of-results-workflow-for-uw-reopening)

File: crontabs/return-of-results [hunk 2]
```diff
@@ -12,4 +12,4 @@ ENVD=/opt/backoffice/id3c-production/env.d
 
 
 # Generate data file and PDFs for returning SCAN IRB results via UW Lab Med's SecureLink portal
```
Comment (kairstenfay) [inline crontabs/return-of-results:14]: TODO update this language

File: bin/return-of-results/generate-results-csv (hunk 1 above)
Comment (kairstenfay) [reply bin/return-of-results/generate-results-csv:26]: Wait, this extra export script will be added when the new REDCap project has been finalized. No change is needed for just doing a rebranding.

File: crontabs/Makefile [hunk 3]
```diff
@@ -2,7 +2,7 @@ SHELL := /bin/bash -euo pipefail
 
 crontabs := \
 	id3c-production \
-	scan-return-of-results \
+	return-of-results \
```
Comment (tsibley) [inline crontabs/Makefile:5]: Note that the installed copy of the old crontab, `/etc/cron.d/backoffice-scan-return-of-results`, will need to be manually deleted immediately before or after deploy.

**Review by tsibley — APPROVED** [review]

**Review by kschwabefry — APPROVED** [review]

**Comment by kairstenfay** [comment]
Rebased onto master.

File: crontabs/Makefile (hunk 3 above)
Comment (kairstenfay) [reply crontabs/Makefile:5]: Done!

**Pull request merged**